### PR TITLE
feat(argo-workflows): Added commonLabels

### PR DESCRIPTION
**Pull request by kvanzuijlen**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-workflows/templates/_helpers.tpl [hunk 1]
```diff
@@ -79,6 +79,9 @@ helm.sh/chart: {{ include "argo-workflows.chart" .context }}
 {{ include "argo-workflows.selectorLabels" (dict "context" .context "component" .component "name" .name) }}
 app.kubernetes.io/managed-by: {{ .context.Release.Service }}
 app.kubernetes.io/part-of: argo-workflows
+{{- with .Values.commonLabels }}
+{{ toYaml .}}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (kvanzuijlen) [reply charts/argo-workflows/templates/_helpers.tpl:84]: Thanks for checking it this quickly! I'll take a look!

[the remark this answers is not in the document]
Comment (kvanzuijlen) [reply charts/argo-workflows/templates/_helpers.tpl]: Sorry about that! I didn't know this functionality yet 😄. TIL

**Review by yu-croco — APPROVED** [review]
Thank you for your contribution. LGTM.

**Comment by kvanzuijlen** [comment]
> Thank you for your contribution. LGTM.

Thank you guys for all the great work!

**Review by jmeridth — APPROVED** [review]

**Pull request merged**